tools/hive/kerberos/scripts/generate_configs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Generate kerberos versions of Hadoop config files by taking the unkerberized versions
+# and tacking on the associated kerberos properties.
+# Assumption is that these two sets of properties do not overlap.
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+for FILE_BASE in core-site hdfs-site hive-site yarn-site; do
+    COMBINED_FILE="../templates/${FILE_BASE}.xml.template"
+    echo "Generating config file: kerberos/templates/${FILE_BASE}.xml.template"
```

Comment:
Not a blocker, but if we were to use Python for this, we could combine the XML as a structured document?
(see for example: https://github.com/mesosphere/dcos-commons/blob/master/frameworks/hdfs/tests/test_overlay.py#L65)